src/openforms/js/components/admin/form_design/registrations/ms_graph/MSGraphOptionsForm.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import OptionsConfiguration from 'components/admin/form_design/registrations/shared/OptionsConfiguration';
+import {filterErrors} from 'components/admin/form_design/registrations/shared/utils';
+import Fieldset from 'components/admin/forms/Fieldset';
+import {
+  ValidationErrorContext,
+  ValidationErrorsProvider,
+} from 'components/admin/forms/ValidationErrors';
+
+import {DriveID, FolderPath} from './fields';
+
+const MSGraphOptionsForm = ({name, label, formData, onChange}) => {
+  const validationErrors = useContext(ValidationErrorContext);
+  const relevantErrors = filterErrors(name, validationErrors);
+  return (
+    <OptionsConfiguration
+      name={name}
+      label={label}
+      numErrors={relevantErrors.length}
+      modalTitle={
+        <FormattedMessage
+          description="Demo registration options modal title"
+          defaultMessage="Plugin configuration: demo"
+        />
+      }
+      initialFormData={{extraLine: '', ...formData}}
```

Comment:
I think the `extraLine` property is a leftover from the DemoOptionsForm?
This is also defined in the `propTypes`